### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by xiaolou86**

### CODE REVIEW

File: protocol/x/clob/keeper/uncommitted_stateful_order_state.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func (k Keeper) SetUncommittedStatefulOrderCount(
 	)
 }
 
-// MustAddUncommittedStatefulOrderPlacement adds a new order placemenet by `OrderId` to a transient store and
+// MustAddUncommittedStatefulOrderPlacement adds a new order placements by `OrderId` to a transient store and
```
Comment (lcwik) [inline protocol/x/clob/keeper/uncommitted_stateful_order_state.go:101]:
```suggestion
// MustAddUncommittedStatefulOrderPlacement adds a new order placement by `OrderId` to a transient store and
```

**Review by lcwik — APPROVED** [review]

**Comment by lcwik** [comment]
@xiaolou86 , you need to sign your commits otherwise this change can't be merged.

`The base branch requires all commits to be signed.`

There are many ways to setup commit signing, see https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits for an example.

**Comment by BrendanChou** [comment]
Closing in favor of scalable solution
https://github.com/dydxprotocol/v4-chain/pull/591

**Comment by xiaolou86** [comment]
It is ok

**Comment by xiaolou86** [comment]
I opened a new PR, please help to review.
#612

**Pull request closed without merging**